vanhouse/src/components/UserList.js

```diff
@@ -45,12 +60,42 @@ function UserList({ setQuery, setUserId }) {
 
 
     return (
-        <div className="user-box">
-            <h2>User List</h2>
-            <Container fluid>{list}</Container>
+        <div>
 
-            <div className="cancel-btn">
-                <button type="button" onClick={Cancel}>Cancel</button>
+            <div className="user-box user_scroll_div d-none d-md-block">
```

Comment:
@tyxxxx In this div there are some `class` attributes, in react we should use className.